vendor/bat-native-ads/src/bat/ads/internal/ad_events/inline_content_ads/inline_content_ad_event_handler.h

```diff
@@ -3,28 +3,30 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-#ifndef BRAVE_VENDOR_BAT_NATIVE_ADS_SRC_BAT_ADS_INTERNAL_AD_EVENTS_INLINE_CONTENT_ADS_INLINE_CONTENT_AD_H_
-#define BRAVE_VENDOR_BAT_NATIVE_ADS_SRC_BAT_ADS_INTERNAL_AD_EVENTS_INLINE_CONTENT_ADS_INLINE_CONTENT_AD_H_
+#ifndef BRAVE_VENDOR_BAT_NATIVE_ADS_SRC_BAT_ADS_INTERNAL_AD_EVENTS_INLINE_CONTENT_ADS_INLINE_CONTENT_AD_EVENT_HANDLER_H_
+#define BRAVE_VENDOR_BAT_NATIVE_ADS_SRC_BAT_ADS_INTERNAL_AD_EVENTS_INLINE_CONTENT_ADS_INLINE_CONTENT_AD_EVENT_HANDLER_H_
 
 #include <string>
 
 #include "base/observer_list.h"
-#include "bat/ads/internal/ad_events/inline_content_ads/inline_content_ad_observer.h"
+#include "bat/ads/internal/ad_events/inline_content_ads/inline_content_ad_event_handler_observer.h"
 #include "bat/ads/public/interfaces/ads.mojom.h"
 
 namespace ads {
 
 struct InlineContentAdInfo;
 
-class InlineContentAd final : public InlineContentAdObserver {
+namespace inline_content_ads {
+
+class EventHandler final : public EventHandlerObserver {
```

Comment:
maybe `InlineContentAdEventHandler` would be better? It also would match with file name.

Reply:
Each handler has namespace for ad type which matches file name. Thanks